src/style-spec/reference/v8.json

```diff
@@ -1972,6 +2001,10 @@
         "doc": "Gets the current zoom level.  Note that in style layout and paint properties, [\"zoom\"] may only appear as the input to a top-level [\"curve\"] expression.",
         "group": "Zoom"
       },
+      "heatmap-density": {
+        "doc": "Gets the density of a pixel in a heatmap layer. Can only be used in `heatmap-color` property.",
```

Comment:
As a lay reader, I’d have little idea what it means for a pixel to have density – other than [pixel density](https://en.wikipedia.org/wiki/Pixel_density), which is a different concept. Any idea how to explain density in the context of a heatmap?